workflow/artifacts/s3/s3.go

```diff
@@ -102,6 +102,22 @@ func loadS3Artifact(s3cli argos3.S3Client, inputArtifact *wfv1.Artifact, path st
 	if origErr == nil {
 		return true, nil
 	}
+
+	if strings.Contains(origErr.Error(), "fileName is a directory.") {
```

Comment:
Is "fileName is a directory" an error message from S3?

Reply:
see https://github.com/minio/minio-go/blob/ab1c2fe050a4dd502c863178696a54c44610ca82/api-get-object-file.go#L45